clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5428,7 +5428,7 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     // only be used for SPIR/SPIR-V based targets.
     if (Triple.isSPIROrSPIRV())
       if (Args.hasFlag(options::OPT_fsycl_instrument_device_code,
-                       options::OPT_fno_sycl_instrument_device_code, true))
+                       options::OPT_fno_sycl_instrument_device_code, false))
```

Comment:
I would expect some LIT test failures that will require changes to pass `-fsycl-instrument-device-code` to match expectations.